Neos.Flow/Classes/Mvc/ActionResponseInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace Neos\Flow\Mvc;
+
+use Psr\Http\Message\UriInterface;
+
+interface ActionResponseInterface
```

Comment:
Why this new interface?

Reply:
So you can dev against the interface, as the implementation right now carries all the bagage of the old HTTP Response.

Reply:
Mh. I think that's counter productive TBH (but that might be related to my jetlag *g). Lets talk about this tomorrow if you wish!?

Reply:
Of course, I was hesitant as well, but I felt it helped to point to some (interface) definition that people can code against until the major. I don't think we absolutely need it, but for a dev not as close into the process I am sure it will help. Let's talk tomorrow :)

Reply:
Yep, see this more as a helper for the time of change. Maybe we can add this in the doc block and already mark it @deprecated?

Basically, the "right" thing would be to `rename` the current `Mvc\ResponseInterface` to `Mvc\ActionResponseInterace` and deprecate it at the same time, but since the signature changes a lot and renaming would be breaking, this is the way to go.

Reply:
OK, it wasn't just the jetlag. I still don't think that we need a new interface here. And an interface that is deprecated right from the start? why?
See main comment for my suggestion on how to approach this